docusaurus.config.js

```diff
@@ -39,18 +39,32 @@ module.exports = {
       {
         id: "optimize",
         path: "optimize",
+        includeCurrentVersion: false,
         routeBasePath: "optimize",
         beforeDefaultRemarkPlugins: [versionedLinks],
-        sidebarPath: require.resolve("./optimize_sidebars.js"),
-        editUrl: "https://github.com/camunda/camunda-platform-docs/edit/main/",
```

Comment:
Per feedback, removing the "Edit this page" link for both optimize (here) & main docs (further down)